test/query.test.js

```diff
@@ -848,6 +848,18 @@ describe('query', () => {
     })
   })
 
+  test.only('create access provider', async () => {
+    const provider = await adminClient.query(
```

Comment:
@marrony I am getting a ` BadRequest: validation failed` error here. I'm using the `adminClient` now since it has an admin key baked in 🤔

Reply:
```
[
  {
    position: [ 'create_access_provider' ],
    code: 'validation failed',
    description: 'document data is not valid.',
    failures: [
      {
        field: [ 'jwks_uri' ],
        code: 'value required',
        description: 'Required value not found.'
      }
    ]
  }
]
```

Reply:
@marrony Looks like on the Core side it's called `jwks_uri` whereas the ticket I implemented had it listed as `jwks_url`. I'll updated the field and confirm that fixes the issue 👍